hydro_lang/src/live_collections/boundedness.rs

```diff
@@ -1,3 +1,6 @@
+//! Type declarations for boundedness markers, which indicate whether a live collection is fully available
+//! in-memory ([`Bounded`]) or asynchronously arriving over time ([`Unbounded`]).
+
```

Comment:
I wonder if "Bounded" is the right word for this?
Bounded makes me think of whether the collection will end at some point in the future, not "is currently in memory" (?)